app.js

```diff
@@ -1,31 +1,28 @@
 const express = require('express');
 const path = require('path');
-const txtgen = require('txtgen');
-
+const textsnippet = require('./api/textsnippetRoutes');
 const app = express();
 
 //body parser Midware
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
-
 //PATH to Public folder
 app.use('/static', express.static(path.join(__dirname, 'public')))
 
-// Create a new (GET METHOD) API route /generateText
-// inside this route generate a text snippet using "txtgen" and send it as a response
-app.get('/generateText', (req, res) => {
-    const textsnippet = txtgen.sentence();
-    res.send(textsnippet)
-})
+// posts a textsnippet to the page.
+app.use('/api/generateText', textsnippet )
```

Comment:
This should be just `/api` because `/generateText` is gonna be configured in `textsnippetRoutes`.